src/test/TestGhoCcipSteward.t.sol

```diff
@@ -252,6 +252,105 @@ contract TestGhoCcipSteward is TestGhoBase {
     );
   }
 
+  function testDisableRateLimitFromNonZeroToZero() public {
```

Comment:
I think we can rename to `testChangeEnabledRateLimit` and test both directions: enabled -> disabled -> enabled

Reply:
as per discussion on slack: it is currently expected behaviour that Steward cannot re-enable disabled rate limits (due to [these range checks](https://github.com/aave/gho-core/blob/3684128be4a1af9d7100b095cd3c6ef6fc0ff62a/src/contracts/misc/GhoCcipSteward.sol#L102-L117) using the current value as max permittable value) and once disabled, the dao can enable rate limits again by directly interacting with the token pool

test for this behaviour added in - [a194c76](https://github.com/aave/gho-core/pull/427/commits/a194c762bba837301997472c2cbe873ce494b756)